core/core.go

```diff
@@ -2,6 +2,12 @@ package core
 
 import "gopkg.in/src-d/core-retrieval.v0/model"
 
+const (
```

Comment:
It's probably better to use a registry for this. Example: https://github.com/src-d/framework/blob/b7e5e412473a20a56c0543f898624f024b164087/lock/common.go#L20
But that would be a different PR.